bindings/python/wscript

```diff
@@ -15,8 +15,11 @@ from waflib.Errors import WafError
 # after = TaskGen.after
 
 # https://github.com/gjcarneiro/pybindgen
-REQUIRED_PYBINDGEN_VERSION = '0.17.0.post41+ngd10fa60'
-REQUIRED_PYGCCXML_VERSION = (0, 9, 5)
+REQUIRED_PYBINDGEN_VERSION = '0.21.0.post20+ng71852b1'
```

Comment:
We are at pybindgen 0.22.0 now.

Reply:
Sir, should I then update this to 0.22.0 for now ?